utils/apk/Makefile

```diff
@@ -0,0 +1,75 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=apk
+PKG_VERSION:=2.12.1
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=apk-tools-v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://gitlab.alpinelinux.org/alpine/apk-tools/-/archive/v$(PKG_VERSION)
+PKG_HASH:=b191dbd5019e8933b78b66779412e901e9b88fb12c460a22995e342b3efe83eb
+
+PKG_MAINTAINER:=Paul Spooren <mail@aparcar.org>
+PKG_LICENSE:=GPL-2.0-only
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/apk-tools-v$(PKG_VERSION)
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/apk
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=apk package manager
+  DEPENDS:=+zlib +libopenssl @!arc
+  URL:=$(PKG_SOURCE_URL)
+endef
+
+define Package/alpine-keys
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Alpine apk public signing keys
+  DEPENDS:=apk
+endef
+
+define Package/alpine-repositories
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Official Alpine repositories
+  DEPENDS:=apk
+endef
+
+MAKE_FLAGS += LUA=no
+
+define Package/apk/install
+	$(INSTALL_DIR) $(1)/lib/apk/db
+
+	$(INSTALL_DIR) $(1)/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/sbin/apk $(1)/bin/apk
+
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/lib/* $(1)/usr/lib/
```

Comment:
I assume this library is used by nothing else. Is it easy to statically link?

Reply:
Good call, I guess I could add a make var

https://gitlab.alpinelinux.org/alpine/apk-tools/-/blob/master/Makefile#L46

Reply:
I looked into statically building it but it seem to pull in openssl as well. I'd avoid that for now and just ship it as an external lib.